scipy/_lib/_testutils.py

```diff
@@ -76,7 +76,8 @@ def __call__(self, label="fast", verbose=1, extra_argv=None, doctests=False,
         pytest_args = ['--showlocals', '--tb=short']
 
         if doctests:
-            raise ValueError("Doctests not supported")
+            # raise ValueError("Doctests not supported")
+            pytest_args += ["--doctest-modules", "--ignore=scipy/interpolate/_interpnd_info.py"]
```

Comment:
@ev-br can't that `_interpnd_info.py` file simply be deleted? It isn't used at all and hasn't seen any nontrivial change since it was written 13 years ago says `git lg --follow scipy/interpolate/_interpnd_info.py`.

Reply:
tl;dr I'd rather keep it.

I think this file implements (a part) of computations of coefficients used in https://github.com/scipy/scipy/blob/main/scipy/interpolate/interpnd.pyx#L392
So it's a form of documentation, an addition to the long comment at https://github.com/scipy/scipy/blob/main/scipy/interpolate/interpnd.pyx#L443, similar to what we have in scipy.special, https://github.com/scipy/scipy/tree/main/scipy/special/_precompute